changelog.txt

```diff
@@ -1,6 +1,7 @@
 *** WooCommerce Subscriptions Core Changelog ***
 
 = 7.3.0 - 2024-xx-xx =
+* Fix - Removing the new "wc-checkout-draft" status from the method used to retrieve related orders.
```

Comment:
```suggestion
* Dev - Introduce new parameter to WC_Subscription::get_last_order() to enable filtering out orders with specific statuses.
```

Given this PR has changed to just introduce a new param that allows callers to exclude specific statuses, the original changelog entry no longer applies.